docker-compose.test.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:latest
```

Comment:
Maybe it's better to use compose `extends` directive here to reuse our base DB and just patch the volume to isolate development and testing data

Reply:
https://docs.docker.com/compose/extends/#extending-services

Reply:
In docker-compose 3 doesn't exist https://docs.docker.com/compose/extends/#extending-services

Reply:
It was deleted from v3, it's true, but the same than the `depends_on`.

From the docs: is ignored in v3 when deploying in swarm mode.

Btw, this is just a proposal :)

Reply:
I tried before and it fails. You cannot use it.

Reply:
Another option is to downgrade to v2 API, v3 is ideal for kubes and swarm